queue-manager/rango-preset/src/helpers.ts

```diff
@@ -351,15 +351,20 @@ export function updateSwapStatus({
     const failureType = mapAppErrorCodesToAPIErrorCode(errorCode);
     updatedResult.failureType = failureType;
 
+    // If trace of error was available, we will send it to the api (except user rejection)
+    const errorReasonForAPI =
+      errorCode !== 'USER_REJECT' &&
+      trace?.message &&
+      typeof trace?.message === 'string'
+        ? trace?.message
+        : errorReason || '';
```

Comment:
```suggestion
    const errorReasonForAPI =
      errorCode !== 'USER_REJECT' &&
      trace?.message &&
      typeof trace?.message === 'string'
        ? trace?.message
        : errorReason || '';
```

Reply:
Feel free to remove these optional chains

Reply:
done